qcodes/instrument_drivers/Minicircuits/USB_SPDT.py

```diff
@@ -66,8 +66,12 @@ def __init__(self, name: str, driver_path: Optional[str] = None,
                 the dll to open properties and check the 'unblock' checkmark
                 in the bottom. Check that your python installation is 64bit."""
             )
-        import mcl_RF_Switch_Controller64
-        self.switch = mcl_RF_Switch_Controller64.USB_RF_SwitchBox()
+        try:
+            import mcl_RF_Switch_Controller64 as mw_driver
```

Comment:
@ThorvaldLarsen when using the net45 one, don't you also need to pass the driver_path explicitly, so that it takes the passed path to DLL? what i have in mind is that we could take the suggested simple solution, and first try importing the net45 driver from the net45 dll and, if that fails, fall back to the "64" one.